### PR TITLE
Added feedback copy as per Howard.

### PR DESCRIPTION
**Pull request by dstarr**

### CODE REVIEW

File: azure-industry-docs/financial/data-mgmt-in-banking-overview.md [hunk 1]
```diff
@@ -199,7 +200,9 @@ and recovery services.
 
 ### Azure Data Box
 
-Another service in preview is focused on migrating very large volumes of data to
+Often, banks will have terabytes, if not petbytes, of information to bring into Azure. Luckily thes data stores in Azure are very elastic and highly scalable.
```
Comment (scseely) [inline azure-industry-docs/financial/data-mgmt-in-banking-overview.md]: petabytes, not 'petbytes'

Comment (scseely) [reply azure-industry-docs/financial/data-mgmt-in-banking-overview.md]: Also, "these" not "thes"

File: azure-industry-docs/financial/data-mgmt-in-banking-overview.md [hunk 2]
```diff
@@ -210,8 +213,15 @@ and secured via standard256-AES encryption. Once the data is on the appliance,
 it is shipped back to the Azure Data Center where the data is hydrated in Azure.
 The device is then securely erased.
 
+## Azure Information Protection
+
+Azure Information Protection (AIP) is a cloud-based solution helping organizations to classify, label, and protect its documents and emails. This can be done automatically by administrators who define rules and conditions, manually by users, or a combination where users are given recommendations.
+
 ## Data services
 
+Banks struggle with Master Data Management, meta data conflicting due to disparate core banking systems
```
Comment (scseely) [inline azure-industry-docs/financial/data-mgmt-in-banking-overview.md]: add a comma after "core banking systems"

File: azure-industry-docs/financial/data-mgmt-in-banking-overview.md [hunk 3]
```diff
@@ -210,8 +213,15 @@ and secured via standard256-AES encryption. Once the data is on the appliance,
...
+and data coming from origination systems, onboarding systems, offers management systems, CRM systems, and more. Azure has tools to help mitigat tehse and other commonly occuring data issues.
```
Comment (scseely) [inline azure-industry-docs/financial/data-mgmt-in-banking-overview.md]: Spelling: "mitigate these"

**Comment by dstarr** [comment]
CLosing to open a different PR

**Pull request closed without merging**